src/content/en/2020/capabilities.md

```diff
@@ -0,0 +1,443 @@
+# Capabilities
```

Comment:
The title is auto-generated based on YAML metadata, which also includes info for authors, reviewers, analysts, etc.

Please see https://raw.githubusercontent.com/HTTPArchive/almanac.httparchive.org/main/src/content/en/2019/performance.md for an example of the metadata to include.

cc @bazzadp

Reply:
Almost done, I just need the ID of the new Discourse thread for this chapter.

Reply:
Let’s use the same id as last year for now.

Reply:
Capabilities is a new chapter, so there's no thread from last year.

Reply:
Ah silly me. Go with 1981 for now which is the 2020 announcement.